tracee-rules/output.go

```diff
@@ -37,6 +38,7 @@ func setupTemplate(inputTemplateFile string, clock Clock) (*template.Template, e
 	case inputTemplateFile != "":
 		return template.New(filepath.Base(inputTemplateFile)).
 			Funcs(funcMap).
```

Comment:
NB: I'm still keeping this in because to my knowledge there's no way to inject a custom time interface into sprig which makes it untestable for our most common case of template (simple and verbose) where we need the current time.

Reply:
why do we need to test it specifically? if we want to test that the templating engine works, can we  craft a simpler template for the test?

Reply:
Yes fair point. I wanted to test the default output (without any templates, which includes a timestamp). I found a another way to do it https://github.com/aquasecurity/tracee/pull/630/commits/bf6a78f87e6175c2c1669448ccb383032c1b38de and cleaned this up.